Stories/Readme.md

```diff
@@ -0,0 +1,52 @@
+# Welcome and Thanks!
+
+We are trying to develop an online data analysis game for citizens. The goal is to give citizens the opportunity to learn more about DNA sequencing and microbiome data. 
+To make it more fun we want to tell the game in a story. We have a few story ideas but we are not sure which is the most capturing. These short stories should get people interested in the game. It would be awesome if you could give us feedback on the stories below. 
+
+
+
+### Story 1: Alien discovering the earth
```

Comment:
Thanks for this hint. You are right maybe it would be better to have a story more tailored to the microbiome. @masako-kaufmann maybe you, or we together can work on this. I changed part of the end to make it more general. But of cores, we can also go back to how it was.